### ISSUE TITLE
Bump celery version

### ISSUE THREAD

**Issue by tayloramurphy** [snowplow/snowplow-python-tracker#198]
The required version of celery is currently at 3.1.11 which is almost 4 years at this point. Similar to #195 , it's causing problems managing virtual environments. I use [pipenv](https://github.com/pypa/pipenv) and I'm unable to generate a lockfile because of this dependency.

**Comment by BenFradet** [comment]
Unfortunately, we haven't had a lot of cycles to dedicate to this tracker lately, a PR would be most welcome :+1:

**Comment by tayloramurphy** [comment]
Maybe I'm doing something completely wrong but I'm having a bear of a time even getting the test suite up and running. I made a note in [this comment](https://github.com/fishtown-analytics/dbt/issues/692#issuecomment-372450679) w/in the dbt repo about my difficulties and the changes I've had to make to get it to even do something. 

The last thing I ran into was redis wasn't running. At that point I ran out of time to try and poke at this some more. :(
